docs/FAQs.md

```diff
@@ -47,3 +47,91 @@ To enable `forceAjcCompile` you need to use following `aspectj-maven-plugin` con
 </configuration>
 ```
 
+## How can I use Powertools for AWS Lambda (Java) with the AWS CRT HTTP Client?
+
+Powertools uses the `url-connection-client` as the default http client. The `url-connection-client` is a lightweight http client, which keeps the impact on Lambda cold starts to a minimum. 
+With the [announcement](https://aws.amazon.com/blogs/developer/announcing-availability-of-the-aws-crt-http-client-in-the-aws-sdk-for-java-2-x/) of the `aws-crt-client` a new http client has been released, which offers faster SDK startup time and smaller memory footprint. 
+
+Unfortunately, replacing the `url-connection-client` dependency with the `aws-crt-client` will not immediately improve the lambda cold start performance and memory footprint, 
+as the default version of the dependency contains low level libraries for all target runtimes (linux, macos, windows, etc).  
+
+Using the `aws-crt-client` in your project requires the exclusion of the `url-connection-client` transitive dependency from the powertools dependency. 
+
+```xml 
+<dependency>
+    <groupId>software.amazon.lambda</groupId>
+    <artifactId>powertools-parameters</artifactId>
+    <version>1.18.0</version>
+    <exclusions>
+        <exclusion>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>url-connection-client</artifactId>
+        </exclusion>
+    </exclusions>
+</dependency>
+```
+Next to adding the `aws-crt-client` and excluding the `aws-crt` dependency (contain all runtime libraries) it's required to set a specific runtime version of the `aws-crt` dependency by specifying the classifier for your specific target runtime.
+Specifying the specific target runtime makes sure all other target runtimes are excluded from the jar file, which will result in the benefit of improved cold start times.   
+
+```xml
+
+<dependencies>
+    <dependency>
+        <groupId>software.amazon.awssdk</groupId>
+        <artifactId>aws-crt-client</artifactId>
+        <version>2.23.21</version>
+        <exclusions>
+            <exclusion>
+                <groupId>software.amazon.awssdk.crt</groupId>
+                <artifactId>aws-crt</artifactId>
+            </exclusion>
+        </exclusions>
+    </dependency>
+
+    <dependency>
+        <groupId>software.amazon.awssdk.crt</groupId>
+        <artifactId>aws-crt</artifactId>
+        <version>0.29.9</version>
+        <classifier>linux-x86_64</classifier>
+    </dependency>
+</dependencies>
+```
+
+After configuring the dependencies it's required to specify the aws sdk http client. 
+Most modules support a custom sdk client by leveraging the `.withClient()` method on the for instance the Provider singleton:
```

Comment:
something missing here...

Reply:
I've tried to clarify it a bit more. I noticed I focussed too much on the parameters module, but other modules also use the url-connection client. Turned it into an example for a module and described that it depends on the PT module in use how to further configure the http client.